builtin/credential/ldap/backend.go

```diff
@@ -118,9 +118,25 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username stri
 	if cfg.BindDN != "" && cfg.BindPassword != "" {
 		if err := c.Bind(cfg.BindDN, cfg.BindPassword); err != nil {
 			if b.Logger().IsDebug() {
-				b.Logger().Debug("error while attempting to re-bind with the BindDN User", "error", err)
+				b.Logger().Debug("error while attempting to re-bind with the BindDN User; trying with new connection", "error", err)
+			}
+			// Create a new client object to work around an issue with
```

Comment:
Can we link the issue in the comment here, which might potentially in future lead to removal of this block of code?